tests/jballerina-unit-test/src/test/resources/test-src/types/map/map-value.bal

```diff
@@ -1,3 +1,15 @@
+public type Animals Map;
+
+public type Map map<string>;
+
+function testUpdateMapValue() {
+    Animals animals = {};
+    foreach var [name, age] in [["animal1", "Cat"], ["animal2", "Dog"]] {
+        animals[name] = age;
+    }
+    assertEquality({"animal1":"Cat", "animal2":"Dog"}, animals);
+}
```

Comment:
Why are we adding at the top?

Reply:
There is no special reason, due to the adding of type definitions, I have added that at the top.

Reply:
We can add it at the bottom, will maintain some level of increasing complexity.

Reply:
Updated.